### ISSUE TITLE
Linux build missing in 1.1 release

### ISSUE THREAD

**Issue by meehow** [mycitadel/mycitadel-desktop#40]
## Operating system
- [x] Linux (all distros)
- [ ] MacOS
- [ ] Windows

## Problem description
```
$ ./mycitadel-linux-amd64 
bash: ./mycitadel-linux-amd64: cannot execute binary file: Exec format error

$ file mycitadel-linux-amd64
mycitadel-linux-amd64: Mach-O 64-bit x86_64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|PIE|HAS_TLV_DESCRIPTORS>
```

Looks like there is Mac binary uploaded instead of Linux binary

**Comment by SkanderHelali** [comment]
Not sure if this was already resolved but I can run the 1.1 release on Linux just fine.

`mycitadel-linux-amd64: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=9ced5e2591cf494222eb4159ce3843db3b9e5fe9, for GNU/Linux 3.2.0, with debug_info, not stripped`

**Comment by dr-orlovsky** [comment]
Yep, this was already resolved
